### PR TITLE
Making navbar for portfolio website

### PR DESCRIPTION
**Pull request by sonuengineer**
- Adding a nav bar section on a website
- creating a headline section..
- adding social media

### CODE REVIEW

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @sonuengineer, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, without a short Pull Request summary explaining in detail the changes added to the project. Kindly edit this and ask again for a review. :pray: :pray:  
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   
+    <header id="nav-bar">
+        <a href="#"><h1>
```
Comment (Nyame-Wolf) [inline index.html]: Good job making the logo a link. I suggest you change it from a `<h1>` element to just a link(no need to put anything can be` <a>Sonu</a>` ). You can only have one `<h1>` element in a document for SEO optimization and screen readers.
read more on that [here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              Sonu
+            </h1></a>
+            <desktop id="header-desktop">
+            <h2 id="portfolio">Portoflio</h2>
+            <h3 id="about-header">About</h3>
+            <h4 id="contact-header">Contact</h4>
+            <img src="images/letter-icon.svg" id="letter-icon">
```
Comment (Nyame-Wolf) [inline index.html]: This should have a display of hidden for mobile. Currently the `about` is visible. I suggest you remove them for now or set their display to none using `display:none`. As per the figma design the only two things visible are the logo and hamburger icon.
Screenshot 2022-10-29 at 16 06 59
This is you page currently
Screenshot 2022-10-29 at 16 12 44

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+           <a href="#"><img src="images/Union.png" id="menu"></a>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest adding both the logo and this hamburger icon to a nav element `<nav>`.The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+}
+
+#menu {
+  margin-right: 20px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+#home {
+  background-image: url(images/Group.svg);
```
Comment (Nyame-Wolf) [inline style.css:40]: Great job using this as a background image.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        <h1>
+            Hey There. I'm Sonu
+        </h1>
+        <h2>
+            I’am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+        </p>
```
Comment (Nyame-Wolf) [inline index.html:38]: I suggest either giving your paragraph a width  Currently your paragraph and the background image are overrunning each other for the width allowed for mobile.
Screenshot 2022-10-29 at 16 12 44
Screenshot 2022-10-29 at 16 21 42

See the figma design:
Screenshot 2022-10-29 at 16 06 59

File: index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      </section>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly add the social media icons to your page just as they are in the figma design. This is the figma design with the icons:
Screenshot 2022-10-29 at 16 06 59

Your page does not have these
Screenshot 2022-10-29 at 16 12 44

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @sonuengineer  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Linters Passing☑️ 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by sonuengineer** [comment]
I did all chamges

**Review by Debas-31 — APPROVED** [review]
## Hi @sonuengineer  :wave, 

# Status: Approved :green_circle: : 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
 

## Highlights
- Linters setup :heavy_check_mark:
- Github flow :heavy_check_mark: 
- Project working :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**